src/components/GlobalFooter/GlobalFooter.js

```diff
@@ -24,6 +29,12 @@ const GlobalFooter = () => {
         </div>
         <div className={styles.social}>
           <h3>Follow Us</h3>
+          <div className={styles.socialMediaIcons}>
+            <img src={facebook} alt = "facebook-icon"/>
```

Comment:
My only comment is HTML syntax calls for removing spaces when declaring HTML attributes, and leaving a space between the last attribute and the forward slash in self-closing HTML elements.

Like this `<img src={facebook} alt="facebook-icon" />`